src/__test__/components/data-management/ProjectMenu.test.jsx

```diff
@@ -35,7 +47,18 @@ const defaultAPIResponse = generateDefaultMockAPIResponses(
   experimentWithSamplesId,
   projectWithSamplesId,
 );
-
+const responses = _.merge(defaultAPIResponse, {
+  [`/v1/access/${fake.EXPERIMENT_ID}-1`]: () => Promise.resolve(new Response(JSON.stringify([{
+    name: 'Bob',
+    email: 'bob@bob.com',
+    role: 'explorer',
+  },
+  {
+    name: 'Mocked User',
+    email: 'mock@user.name',
+    role: 'owner',
+  }]))),
+});
```

Comment:
Can you format this piece of code with better identation so it's easier to read?